api/swagger.json

```
@@ -0,0 +1,1304 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "description": "This is the API specification for Chapter",
+    "version": "1.0.0",
```

This would not be a stable (1.0) version of the API, so let's use a minor.
```suggestion
    "version": "0.1.0",
```